src/styles/blocks/card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  @include flex($direction: column, $items: center);
+
+  box-sizing: border-box;
+  max-width: 200px;
+  width: 100%;
+  border: 1px solid $light-white-color;
+  border-radius: $default-border-radius;
+  background-color: $white-color;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    @include font(12px, 500, 18px);
+
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    @include font($line-height: 14px, $color: $secondary-color);
```

Comment:
Ensure that the `font` mixin and the `$secondary-color` variable are defined and imported correctly. Missing definitions could lead to styling issues.